addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/cloudapi/VoiceRSSCloudImplementation.java

```diff
@@ -48,11 +47,12 @@
     private static Set<String> supportedVoices = getSupportedVoices();
 
     /**
-     * Will support only "MP3" for the moment.
+     * Will support only "MP3" and "OGG" for the moment.
      */
     private static Set<String> getSupportedAudioFormats() {
         Set<String> formats = new HashSet<String>();
-        formats.add(AudioFormat.CODEC_MP3);
+        formats.add("MP3");
```

Comment:
It feels strange to switch from an smarthome constant to a string, what is the reason?

Note: It seems OGG is only a container and VORBIS is the real codec?